tests/test_stats.py

```diff
@@ -75,13 +74,13 @@ def test_sum(
 @pytest.mark.parametrize("dtype", [np.float32, np.float64])  # random only supports float
 def test_sum_benchmark(
     benchmark: BenchmarkFixture,
-    array_cls_name: str,
+    array_type: ArrayType,
     axis: Literal[0, 1, None],
     dtype: type[np.float32 | np.float64],
 ) -> None:
     try:
-        shape = (1_000, 1_000) if "sparse" in array_cls_name else (100, 100)
-        arr = random_array(array_cls_name, shape, dtype=dtype)
+        shape = (1_000, 1_000) if "sparse" in array_type.mod else (100, 100)
+        arr = array_type.random(shape, dtype=dtype)
     except NotImplementedError:
```

Comment:
Re: `random` comment, would be interesting to see how this plays out with dask esp. multi-processed but not sure how much sense it makes with potential overhead of processes

Reply:
IDK if codspeed supports that also

Reply:
Fair, so might be worth just making that clearer in the interim i.e., it's not that it isn't implented but XYZ

Reply:
where should I make that clearer? comment or error message or … ?

Reply:
Ah I understand now. done